src/client/components/ResultMessageModalWindow/FinalMessage.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function FinalTaskMessage({ correctAnswer, checkpoints }) {
+  if (correctAnswer > 3) {
+    return (
+      <div className="final_message-component">
+        <p>
+          You have now answered correct on {correctAnswer} out of 5 questions
+          right.
+          <br />
+          You&#39;re a smart pirate.
```

Comment:
Aren't you suppose to put a white skull icon after "you are a smart pirate."

Reply:
yes. but I searched on HTML Unicode of it and how can I write it but cant found it ... I will search again after you give me the name of it  (a white skull icon) thanks

Reply:
I found it &#9760;
thanks

Reply:
You did good, but it doesn't look like the reference design. I am sure that if you put some container around it you will be able to write some css for it as well but except that i will recommend you putting the actual image here. Sorry for the excess work 😬  but it is needed for your component 🙂